build/scripts/main.js

```diff
@@ -0,0 +1,83 @@
+$(document).ready(function() {
+    UpdateDateTime();
+});
+
+/*
+Load the date and populate the DOM.
+*/
+function LoadDate()
+{
+    // Date
+    var dateCurrentDate = new Date();
+    var dateMonth = getMonthString(dateCurrentDate.getMonth());
+    var dateDay = dateCurrentDate.getDate();
+    var dateString = dateMonth + " " + dateDay;
+    $(".greeting-date").text(dateString);
+    return;
+}
+
+/*
+Load the time and populate the DOM.
+*/
+function LoadTime()
+{
+    var timeCurrent = new Date();
+    var timeHours = timeCurrent.getHours();
+    var timeMinutes = timeCurrent.getMinutes();
+    var timeSeconds = timeCurrent.getSeconds();
+    var timeString = timeHours + ":" + TwoDigitString(timeMinutes) + ":" + TwoDigitString(timeSeconds);
+    $(".greeting-time").text(timeString);
+    
+    return;
+}
+
+/*
+Run LoadDate and LoadTime on an interval
+*/
+function UpdateDateTime()
+{
+    LoadDate();
+    var intervalDate = setInterval(LoadDate, 100);
+    LoadTime();
+    var intervalTime = setInterval(LoadTime, 100);
+}
+
+/*
+Show two digits for minutes and time. 
+
+Source: 
+*/
+function TwoDigitString(_input)
```

Comment:
Nit: I might rename this function to make it more explicit what it does.

Reply:
Fair enough.